.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@ The more detailed you are, the better.**
 Before submitting the PR make sure the following are checked:
```

Comment:
Should we change this to say "before merging the following needs to be completed"?